Source/TcpSocketPlugin/Public/TcpSocketConnection.h

```diff
@@ -72,34 +72,116 @@ class TCPSOCKETPLUGIN_API ATcpSocketConnection : public AActor
 	UFUNCTION(BlueprintPure, meta = (DisplayName = "Append Bytes", CommutativeAssociativeBinaryOperator = "true"), Category = "Socket")
 	static TArray<uint8> Concat_BytesBytes(TArray<uint8> A, TArray<uint8> B);
 
+	/** Converts a byte to an array of bytes */
+	UFUNCTION(BlueprintPure, meta = (DisplayName = "Byte To Bytes", CompactNodeTitle = "->", Keywords = "cast convert", BlueprintAutocast), Category = "Socket")
+	static TArray<uint8> Conv_ByteToBytes(uint8 InByte);
+
+	/** Converts a short to an array of bytes */
+	UFUNCTION(BlueprintPure, meta = (DisplayName = "Short To Bytes", CompactNodeTitle = "->", Keywords = "cast convert", BlueprintAutocast), Category = "Socket")
+	static TArray<uint8> Conv_ShortToBytes(int16 InShort)
```

Comment:
Please move the function definition into .cpp
And do in-place initialization, e.g.
```cpp
return TArray<uint8> { 0, 1, 2 };
```